src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  margin: 0;
+  padding: 0;
+
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+  color: #3d4e61;
+  line-height: 22px;
+}
+
+.search {
+  margin: 20px 8px;
+  position: relative;
+}
+
+.big-search {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  font: inherit;
+  font-size: 16px;
+}
+
+.small-search {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  font: inherit;
+}
+
+.big-search::placeholder,
+.small-search::placeholder {
+  font: inherit;
+}
+
+.big-search,
+.small-search,
+.big-search:hover,
+.small-search:hover,
+.big-search:focus,
+.small-search:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+}
+
+.label-big::before {
+  position: absolute;
+  content: " ";
+  background-image: url(./images/Search.svg);
+  background-size: cover;
+  left: 26px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.big-search:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.label-small::before {
+  position: absolute;
+  content: " ";
+  background-image: url(./images/Search.svg);
+  background-size: cover;
+  left: 13px;
+  top: 15px;
+  width: 11px;
+  height: 11px;
+}
+
+.big-search:hover,
+.small-search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small-search:focus,
+.big-search:focus {
+  font-weight: 700;
+  background-color: #f6f6f7;
+}
```

Comment:
It's better to set bold font-styles for input (not on focus state) and normal for ::placeholder